test/suite/test_tiered19.py

```diff
@@ -196,7 +196,7 @@ def test_ss_azure_file_system(self):
         self.assertEqual(ss.ss_flush_finish(session, azure_fs, 'foobar', 'foobar', None), 0)
 
         # The object exists now.
-        self.assertEquals(azure_fs.fs_directory_list(session, '', prefix_1), [prefix_1 + 'foobar'])
```

Comment:
I'm probably missing something, but why did these `directory_list` calls get a bunch of args removed?

Reply:
Yup this was changed in the https://jira.mongodb.org/browse/WT-10580 where the list objects now lists everything inside the container which is the intention of this test!

Reply:
Gotcha, thanks :-)